tests/sentry/seer/anomaly_detection/test_store_data.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+
+from sentry.seer.anomaly_detection.store_data import fetch_historical_data, format_historical_data
+from sentry.snuba.models import SnubaQuery
+from sentry.testutils.factories import EventType
+from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.utils.snuba import SnubaTSResult
+from tests.sentry.incidents.endpoints.test_organization_alert_rule_index import AlertRuleBase
+
+
+class AnomalyDetectionStoreDataTest(AlertRuleBase):
```

Comment:
If we're `store_event`'ing data, then i think we'll want to inherit from the SnubaTestCase as well

Reply:
I don't think there'd be any data if it wasn't actually stored.. ?